src/components/Testimonial.astro

```diff
@@ -0,0 +1,26 @@
+---
+    export interface Props {
+        mainImageAlt?: string;
+        quote: string;
+        authorInformation: string;
+        authorDescription: string;
+        page:string;
+    }
+
+    const { mainImageAlt = " ", quote, authorInformation, authorDescription, page } = Astro.props;
+---
+
+<section class={`testimonial testimonial--${page}`}>
+    <picture class="testimonial__shape-container">
+        <source media="(min-width: 1024px)" srcset={`/assets/images/${page}/testimonial/testimonial-shape-desktop.svg`}>
+        <img class="testimonial__shape" src={`/assets/images/${page}/testimonial/testimonial-shape-mobile.svg`} alt=" ">
+    </picture>
+    <figure class="testimonial__image-container">
+        <img src={`/assets/images/${page}/testimonial/testimonial.png`} alt={mainImageAlt} class="testimonial__image">
```

Comment:
This page Prop makes sense, maybe we need to different ones, page and shapeType based on my previous comment.